lib/restrictions.js

```diff
@@ -74,9 +74,9 @@ export async function insert({restriction} = {}) {
 /**
  * Updates an existing restriction, replacing it entirely.
  *
- * @param {object} options - Options to use.
- * @param {object} options.restriction - An application-specific string that
- *   unambiguously identifies a particular entity to the application.
+ * @param {object.<symbol, object>} [options = {}] - Options to use.
```

Comment:
This really can't just be:

```suggestion
 * @param {object} [options = {}] - Options to use.
```

This would cause a linter error? Seems like maybe we need to fix the linting rules here.

@mattcollier -- thoughts on this one?

Reply:
Other than this one item (and its appearance further up), the rest of the PR LGTM.

Reply:
@JSAssassin, I pushed a commit here that moves the `Restriction` definition into its own block. I'm going to approve this PR with that change ... let me know if you object and we can back out the commit and find a different solution.

Reply:
just so you know this is probably an error in the plugin itself. The solution of using a property is nice though. The jsdoc plugin seems to struggle with syntax quite often.

Reply:
@dlongley thanks, LGTM.